### PR TITLE
CI :Edit job

### PR DESCRIPTION
**Pull request by FatimaAlmashhor**
I do this to practice I will merge it and then create develop branch .

### CODE REVIEW

**Review by MuhannedNoman — APPROVED** [review]
LGTM.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+  build-and-test:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: npm install
+            - run: npm run format
+            - run: "[[ -z $(git --no-pager diff -- . ':(exclude)yarn.lock' ':(exclude)yarn.lock') ]]"
```
Comment (FatimaAlmashhor) [inline .circleci/config.yml:14]: how about this line should i change the yarn ? @MuhannedNoman

**Review by FatimaAlmashhor — COMMENTED** [review]
@MuhannedNoman I am afraid to do something stupid .

File: .circleci/config.yml (hunk 1 above)
Comment (MuhannedNoman) [reply .circleci/config.yml:14]: Yes `package-lock.json`

Comment (MuhannedNoman) [reply .circleci/config.yml:14]: Why haven't this been implemented before the merge ?

**Pull request merged**